README.md

```diff
@@ -79,6 +79,45 @@ sudo -u mapped-user aws s3 ls s3://my-custom-mapper/install
 2020-09-11 19:07:41        254 user-role-mapper.properties
 ```
 
+### Integration tests
+- Set the AWS credentials to be available to the environment before running the tests.
```

Comment:
1/ Let's also mention that  users should not run integration test with multi-threading.
2/ Let's also give a brief explanation on the integ test? how does it setup etc.

Reply:
Done.